ui.R

```diff
@@ -125,12 +125,20 @@ ui <- dashboardPage(
       
       tabItem(
         tabName = "tab_dclist",
-        radioButtons(
-          inputId = "input_region",
-          label = "Please choose a region of Hong Kong",
-          choices = sort(unique(data_region))
-        )
+        selectInput(inputId = "input_consituency_en",
+                    label = "Please choose a constituency",
+                    choices = sort(unique(data_master_raw$Constituency_EN))),
+        selectInput(inputId = "input_constituency_zh",
+                    label = "Please choose a constituency",
+                    choices = sort(unique(data_master_raw$Constituency_ZH))),
         
+        column(
+          width = 12,
```

Comment:
Unfortunately, this places a restriction on the `valueBoxOutput()` below so if you have a really long party name, the text flows outside of the valueBox.

Reply:
We can apply some sort of truncation or string replacement to ensure it fits within 12 characters (or cm?). This can be updated as part of the data preparation script.

Reply:
Ahh, nah, rather not do any truncation/string replacement. Should try to keep the full name of the party. I think we need to think of how to get the `valueBox()` to expand and contract according to the text. I think it does so already, but reaches a maximum width. I remember 12 being the maximum width for the shinydashboard so no idea why it's taking up a small width. Maybe we should omit the `width = 12`.